### DISCUSSION TITLE
Huge codebase: How to measure performance delta with native stack (v6)?

### DISCUSSION THREAD

**Discussion by mihir0x69** [react-navigation/react-navigation#10435] [Q&A]
I'm unable to see any gains after switching to native stack navigator (v6). Can anyone provide any pointers on measuring the performance benefits?

**Comment by brandon-gs** [comment]
Did you find something?

**Reply by raajnadar** [reply]
Please try this https://github.com/react-navigation/react-navigation/discussions/10435#discussioncomment-9504438

**Comment by raajnadar** [comment]
Not sure if that will be useful but you guys can try this tool and let us know if that gave some good insights

https://docs.infinite.red/reactotron/plugins/benchmark/
